lambdas/service/app.py

```diff
@@ -341,7 +345,17 @@ def service_config(self) -> ServiceConfig:
 
     @property
     def facets(self) -> Sequence[str]:
-        return sorted(self.service_config.translation.keys())
+        return list(self.field_types_by_filterable_facets.keys())
```

Comment:
Why do we need to couple these two methods?

Reply:
IIRC the filterable facets and the sort facets are the same set of facets.

`app.facets` is used to expose sort facets within the OpenAPI spec, and is interpolated into parts of the documentation. By coupling together we ensure the documentation only shows both filterable and "sort-able" facets that can be used. 

<img width="1275" alt="Screen Shot 2021-12-16 at 9 58 55 AM" src="https://user-images.githubusercontent.com/1947534/146424281-f167c3b3-4af3-40b0-ba4a-4a53ea89d428.png">